### ISSUE TITLE
Feedback from class, 4/19/2022

### ISSUE THREAD

**Issue by bmkuter** [keven-deoliveira/HealthCare-Platform#5]
- Nice login/sign-in page! Good use of external API so you don't need to reinvent the wheel. Is the user ID a hash based on the users Google ID?
- Thank you for images on GitHub readme. Very useful inclusion. 
- Good documentation so far! Helps the reader see not only the full features of the project, but also how to utilise it. 
- Good forms! How are you doing input sanitisation? 
- Is the database hosted locally or on a cloud service?

**Comment by keven-deoliveira** [comment]
> * Nice login/sign-in page! Good use of external API so you don't need to reinvent the wheel. Is the user ID a hash based on the users Google ID?

Yes, the userID is a hash based on their information pulled from the Google API. I figured this would be the easiest way to identify users uniquely.

>     * Thank you for images on GitHub readme. Very useful inclusion.

>     * Good documentation so far! Helps the reader see not only the full features of the project, but also how to utilise it.

>     * Good forms! How are you doing input sanitisation?

From what I understand, Django handles SQL injections already by using an authentication library that authenticates requests and sanitizes them before querying the database. I did some testing and it seemed to be consistent, but there were times where it did not work well. I plan on adding sanitation to the form where I was able to inject.

>     * Is the database hosted locally or on a cloud service?
It is hosted on a cloud service, also on AWS EC2 instance
